src/Adapter/Tax/CommandHandler/BulkUpdateTaxStatusHandler.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Adapter\Tax\CommandHandler;
+
+use PrestaShop\PrestaShop\Core\Domain\Tax\Command\BulkUpdateTaxStatusCommand;
+use PrestaShop\PrestaShop\Core\Domain\Tax\CommandHandler\BulkUpdateTaxStatusHandlerInterface;
+use PrestaShop\PrestaShop\Core\Domain\Tax\Exception\CannotToggleTaxStatusException;
+use PrestaShop\PrestaShop\Core\Domain\Tax\Exception\TaxException;
+use Tax;
+
+/**
+ * Class BulkUpdateTaxStatusHandler handles command which updates Taxes status in bulk action
+ */
+final class BulkUpdateTaxStatusHandler extends AbstractTaxHandler implements BulkUpdateTaxStatusHandlerInterface
+{
+
+    /**
+     * {@inheritdoc}
+     */
+    public function handle(BulkUpdateTaxStatusCommand $command)
+    {
+        foreach ($command->getTaxesIds() as $taxId) {
+            $tax = new Tax($taxId->getValue());
+            $this->assertTaxWasFound($taxId, $tax);
+
+            try {
+                if (!$tax->save()) {
+                    throw new CannotToggleTaxStatusException(
```

Comment:
command/handler says it "updates" but exception says "toggle", i think you should use same naming in both.